packages/typography/src/Link.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useMemo } from 'react';
+import ArrowRightIcon from '@leafygreen-ui/icon/dist/ArrowRight';
+import OpenNewTabIcon from '@leafygreen-ui/icon/dist/OpenNewTab';
+import Box, { ExtendableBox } from '@leafygreen-ui/box';
+import { useBaseFontSize } from '@leafygreen-ui/leafygreen-provider';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { HTMLElementProps, createDataProp } from '@leafygreen-ui/lib';
+import { typeScale1, typeScale2 } from './styles';
+
+const anchorDataProp = createDataProp('anchor-container');
+
+const linkStyles = css`
+  text-decoration: none;
+  color: ${uiColors.blue.base};
+  cursor: pointer;
+
+  &:focus {
+    outline: none;
+  }
+`;
+
+const underline = css`
+  background-repeat: repeat-x;
+  background-position: 0 1em;
+  background-size: 3px 3px;
+
+  ${anchorDataProp.selector}:hover & {
+    background-image: linear-gradient(
+      to right,
+      ${uiColors.gray.light2} 100%,
+      ${uiColors.gray.light2} 0
+    );
+  }
+
+  ${anchorDataProp.selector}:focus & {
+    background-image: linear-gradient(to right, #9dd0e7 100%, #9dd0e7 0);
+  }
+`;
+
+const arrowRightIconPersist = css`
+  transform: translate3d(3px, 0, 0);
+`;
+
+const arrowRightIconHover = css`
+  height: 10px;
+  width: 10px;
+  opacity: 0;
+  transform: translate3d(-3px, 0, 0);
+  transition: all 100ms ease-in;
+
+  ${anchorDataProp.selector}:hover & {
+    opacity: 1;
+    transform: translate3d(3px, 0, 0);
+  }
+`;
+
+const openInNewTabStyles = css`
+  margin-bottom: -1px;
+  margin-left: -1px;
+  margin-right: -2px;
+`;
+
+declare const __TARGET__: 'web' | 'node';
+
+function isServer() {
+  return __TARGET__ === 'node';
+}
+
+const ArrowAppearance = {
+  Hover: 'hover',
+  Persist: 'persist',
```

Comment:
Hey! I noticed while playing with the link that the persistent arrows aren't spaced the same from the text as the hover ones. We might want to apply the same translation we do on hover to the persistent version as well.

Reply:
That's a storybook problem. I had the value `persistant` when the value should be `persist`
Will update storybook to pull from the enum directly